### DISCUSSION TITLE
How to change the color of the background of this goldendict-ng scanning pop-up window dictionary?

### DISCUSSION THREAD

**Discussion by closetruth** [xiaoyifang/goldendict-ng#1354] [Q&A] [answered]
![gold](https://github.com/xiaoyifang/goldendict-ng/assets/155558417/8cd93aff-1754-4da7-9872-8d0f4d9d4483)

**Comment by closetruth** [comment]
I use archlinux

**Comment by xiaoyifang** [comment]
what is the version?
looks quite old

**Reply by closetruth** [reply]
archlinuxcn/goldendict-ng-git 23.12.08.r5279.804aa297-1 [installed]
    The next generation GoldenDict (Supports Qt WebEngine & Qt6).

**Comment by closetruth** [comment, marked as answer]
I found it. It has two backgrounds, one is from goldendict and the other is from dictionary file, so it is modified from goldendict css file and modified from dictionary file.
